.github/workflows/unit-test.yml

```diff
@@ -22,12 +22,12 @@ on:
     paths-ignore:
       - '**/*.md'
       - 'dolphinscheduler-ui'
-      - 'dolphinscheduler-python/pydolphinscheduler'
+      - 'dolphinscheduler-python/pydolphinscheduler/**'
```

Comment:
if this is needed, then the line ☝️ (`'dolphinscheduler-ui'`) might also need `/**`?

Reply:
Good catch 👍 . BTW I just imitate the way `ui` but failed 🤣